recipes/_packaging.rb

```diff
@@ -39,6 +39,9 @@
   package 'zlib-devel'
   # EL 7 split rpm-sign into its own package:  http://cholla.mmto.org/computers/linux/rpm/signing.html
   package 'rpm-sign' if node['platform_version'].satisfies?('>= 7')
+  # EL 6 and later consider glibc-static optional: https://access.redhat.com/solutions/33868
+  # The libhugetlbfs build (analytics) requires it. 
+  package 'glibc-static' if node['platform_version'].satisfies?('>= 6')
```

Comment:
I think we should make the runit cookbook use ian meyer's packagecloud repo for RHEL packages.

https://github.com/hw-cookbooks/runit/issues/88